libcircle.spec

```diff
@@ -66,25 +68,26 @@ A simple interface for processing workloads using an automatically
 distributed global queue.
 
 This package contains development headers and libraries for
-MPICH ibcircle
+MPICH libcircle
 
 %prep
-%setup -q -n %{name}-%{myversion}
+%setup -q
+./autogen.sh
```

Comment:
Is `autogen,sh` really needed for a build from an upstream release?  Autotools based projects typically release with `configure` (and friends) already created.

Reply:
Upstream just put a tag in the distro, and did not create a release build.

Reply:
I see a `configure` in [libcircle-0.3.0.tar.gz](https://github.com/hpc/libcircle/releases/download/v0.3/libcircle-0.3.0.tar.gz).  Is that what you are using or are you using the auto-generated-from-tag `.tar.gz`?

Reply:
The spec file is using the auto-generated-from-tag.tar.gz.  I attempted to use the release urls at first and was getting a 404.

Ah, the release version name in the filename does not have the "v" before the version.